libcudacxx/test/libcudacxx/std/algorithms/alg.nonmodifying/alg.for_each/pstl.for_each.pass.cpp

```diff
@@ -0,0 +1,64 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Q: Would it be possible to implement these lit tests as C2H tests? That would greatly simplify splitting them out for dependency tracking in CI.